CDCgov/NEDSS-Modernization
# CNFT2-1177-delete-static-element

## Description

Delete Static Element API, given a specific ID for a static element on the page.

## Tickets

* [CNFT2-1177](https://cdc-nbs.atlassian.net/browse/CNFT2-1177)

## Steps to verify

1. Using swagger, create a static element of each type
2. Then use that componentID returned as the response to create a request for the delete static element 
3. If you have NBS running locally, you can manually check the database or launch the NBS classic UI and edit the corresponding page to then see whether the static elements created exist


[CNFT2-1177]: https://cdc-nbs.atlassian.net/browse/CNFT2-1177?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
